bitsandbytes/autograd/_functions.py

```diff
@@ -263,6 +263,15 @@ def get_tile_size(self):
         ), f"please find this assert and manually enter tile size for {self.formatB}"
         return (8, 32) if self.formatB == "col_turing" else (32, 32)
 
+    @property
+    def tile_indices(self):
```

Comment:
Why not use `cached_property`?

Reply:
As far as I know, it is [only available](https://docs.python.org/3/library/functools.html#functools.cached_property) in Python 3.8: hence, the code won't work on systems with older Python. Still, thanks for the suggestion — I haven't actually thought of it when working on the PR!